lib/benchee/system.ex

```diff
@@ -139,6 +144,11 @@ defmodule Benchee.System do
 
   def parse_cpu_for(:FreeBSD, raw_output), do: String.trim(raw_output)
 
+  def parse_cpu_for(:Solaris, raw_output) do
```

Comment:
Adding a test may be neat, that's the reason that function is public :) It's also nice for me to have an example of the output: https://github.com/bencheeorg/benchee/blob/main/test/benchee/system_test.exs#L49

Reply:
```
# kstat -j cpu_info:0::brand
[{
        "module": "cpu_info",
        "instance": 0,
        "name": "cpu_info0",
        "class": "misc",
        "type": 1,
        "snaptime": 3916379.363662097,
        "data": {
                "brand": "Intel(r) Xeon(r) CPU E5-2678 v3 @ 2.50GHz"
        }
}]
```